flake8_isort.py

```diff
@@ -1,4 +1,7 @@
 # -*- coding: utf-8 -*-
+from flake8_polyfill import stdin
+stdin.monkey_patch('pycodestyle')
```

Comment:
Yes, I very much would prefer that, so we keep the import side effects away from the code